crates/turborepo-paths/src/absolute_system_path.rs

```diff
@@ -235,6 +235,11 @@ impl AbsoluteSystemPath {
         Ok(AbsoluteSystemPathBuf(cleaned_path))
     }
 
+    pub fn canonicalize(&self) -> Result<AbsoluteSystemPathBuf, PathError> {
+        let canonicalized = self.0.canonicalize_utf8()?;
```

Comment:
We probably can't use this cause we need to use [dunce](https://docs.rs/dunce/latest/dunce/) to avoid UNC paths on Windows.

Reply:
Ah good call, I was looking for `to_realpath`. I'll update it.